__tests__/itinerary.test.js

```diff
@@ -0,0 +1,33 @@
+const Itinerary = require("../src/itinerary");
+const Ship = require("../src/ship");
```

Comment:
You shouldn't be import `Ship` into this file. If you do need to use `Ship` for any reason you should be thinking of ways you can stub/mock it out